src/env.h

```diff
@@ -721,8 +731,6 @@ class Environment {
   std::unique_ptr<inspector::Agent> inspector_agent_;
 #endif
 
-  HandleWrapQueue handle_wrap_queue_;
-  ReqWrapQueue req_wrap_queue_;
```

Comment:
For some reason `Environment::req_wrap_queue()` is completely broken in the cctest build (I tested it on the master branch too), it wasn't returning the right address for `req_wrap_queue_` (it was always off by -0x20 bytes), thus manipulating or accessing this queue was impossible on tests. Probably there's some flag or macro different from the binary build.

As there were no tests using `Environment::req_wrap_queue()` before it wasn't a problem, but with the introduction of `test_node_debug_support.cc` `Environment::req_wrap_queue()` should work, and moving this to the beginning of the class fixed it.

Reply:
Would it good to get to the bottom of that.  Moving data members around without really understanding the problem is not optimal.

Reply:
I spent a few days trying to get to the bottom of it, but without success. Best I can tell is that it looks like some alignment problem and/or some member which is conditionally compiled (either doesn't make complete sense, but was the best I could get after analyzing the problem for some time). I'll open an issue with what I was able to find, as this problem is also present on `master`

Reply:
Found the root of this problem. Solution proposed on: #17461